### ISSUE TITLE
Crash after placing Stabilized Mob Spawner

### ISSUE THREAD

**Issue by UncleSweaty** [AllTheMods/ATM-3#121]
This happened on my locally hosted server. I placed a Draconic tier Stabilized Mob Spawner in my mob grinder room and the game instantly crashed. At the time of placing the block, the gamemode was set to peaceful. The mob spawner was either a Blaze or a Magma Cube. It happened in the overworld. I have placed Draconic tier Stabilized Mob Spawners down before in the same area. I have tried restarting both the client and the server and I still crash when I'm about to enter the game, though the server is unaffected.

Here is the crash log: http://textuploader.com/d685p

**Comment by TheElectricCake** [comment]
Does this crash occur if the game is in non-peaceful mode?

**Comment by UncleSweaty** [comment]
I rolled back to a previous backup and could not replicate the bug. The things present were 9 other Draconic Stabilized Spaweners from Draconic Evolution, a single Mob Grinder from Draconic Evolution, an Environmental Controller from RFTools with No Enderman Teleport Module installed, Item Collector from Cyclic, Experience Solidifier from Actually Additions, some Fast Slime Conveyors on the floor, Signalum Fluxducts form Thermal Dynamics to power the machines, and Signalum Plated Impulse Itemducts from Thermal Dynamics to carry the collected items away, all contained within a 10x8x10 box of Thickened Glass.

I tried replicating the bug on an offline Superflat creative world but experienced no crash.
Thanks for helping.
